### ISSUE TITLE
[Design] - Learning Center MVP - Update search results page to reflect search.gov as data source

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#12863]
Update the design of the search results page to reflect that the description under the title will come from search.gov snippets and aren't something we can control.

**Acceptance Criteria**
- [ ] Update design of desktop search results pages based on above and share with team

**Comment by jenniferlee-dsva** [comment]
@andaleliz - one pertinent simplification out of the user research findings: 

- We are including the content template labels on LC search results display
- We are excluding the article tags on LC search results display (for both keyword search results page and on 'view all' X articles and tagged articles pages)
- Per slack, we are excluding/stripping out text links that may be in the intro text if that part of the text gets pulled into the results display

**Comment by rtwell** [comment]
![image](https://user-images.githubusercontent.com/12564977/91853095-b7d48e00-ec2f-11ea-8eea-26a72a494546.png)

**Comment by andaleliz** [comment]
@rtwell thanks for doing these! These look great. I don't think we need two versions of both, but past experiences tells me  we will need a version of how the page will look when a user is viewing an `articles tagged` or `go to articles [topic]` results page, for QA and likely our developers too. Desktop or mobile version would be fine for those purposes.

I think the articles should be sorted alphabetically for `articles tagged` or `go to articles [topic]` results - what do you think? Most relevant makes sense for a keyword search.

Also, I'd make the `h1` different too, since the user didn't search. Something like `Articles tagged: burials and memorials`.  Not sure if that has been discussed.

Last week, some technical discovery was still needed to determine how the description under the results will be handled. We can update that when we know more.

One small thing I noticed - there's a stray capital `L` in the CTA above the search input.

**Comment by rtwell** [comment]
yes, I have those as well. Do we have separate ticket for those? or would you rather I drop them here?

**Comment by andaleliz** [comment]
This ticket will cover all of them - I saw those in the Sketch file after I posted that comment.  Thanks for doing them!
 
The only thing we need to wait on is the outcome of that technical discovery so no need to post now since they might change again later.  I'm trying to stay on top of that and will keep you posted.

**Comment by andaleliz** [comment]
@jenniferlee-dsva is this ticket still needed?  I had been waiting to do work on this because I needed more details about what is possible with the descriptions - they're really erratic on our own site search, and I'm not sure if we can control that in the learning center.  

I can continue to follow up in order to produce this mock-up if so.

**Comment by andaleliz** [comment]
@jenniferlee-dsva, some updates:

**Keyword search result pages**
Talked to @ncksllvn and team today to get more details about the descriptions under the titles on our keyword search result page. Based on what we know, it seems like we don't have much influence on how those descriptions will appear under the titles.  I can create mock-ups that reflect what we'd like to see, but the reality is that we're at the mercy of the search crawler. Do you still think updated mock-ups would be useful for this screen?  I do think it'd be good to update the results text to include how things are sorted, e.g. `Showing 1-10 of 89 results for "burials" sorted by relevance`, so I could include that in a new mock-up.

**Articles tagged/ View all articles pages**
We have more control over this view, so I can provide UX guidance on a couple of things: 
- I understand the CMS meta-description field is going to be auto-populated w/ the intro text. I would recommend pulling in the first two sentences, but not exceeding 300 characters. If the first 2 sentences are more than 300 characters, we could have an ellipses in something like: `<span aria-hidden="true">. . .</span>` so screenreader users don't have to hear `dot dot dot` every time. Would need to validate that mark-up with a11y but that's a start.
- Articles are sorted by title alphabetically for MVP 1, with the intention of evaluating if this makes sense for people once we have more content categories than just the 2 we are launching with
- We add text to tell users how things are sorted, e.g. `Showing 1-10 of 89 articles in "VA account and profile" sorted alphabetically`

**Comment by andaleliz** [comment]
Also, the CMS team is discussing the meta-description on learning center pages in their meeting tomorrow, asked to be invited.

**Comment by jenniferlee-dsva** [comment]
Hope this answers some of the Qs: 

**9/23 - search results and landing pages -** 
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/status-and-decisions.md#92320

RE what text would display on dynamic user query based search results: Search.gov folks confirmed that they pull  the page content that maps by relevance to the user query. We are okay with this -- for dynamic search results listings -- so we will not populate the meta description field on articles. (This field doesn't improve external organic searches in any case.) 

For what text to display on static 'view all articles in Xcategory' or 'tagged: Y' landing pages - see: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/view-all-articles-in-X-category-landing-pg-requirements.md

**Comment by andaleliz** [comment]
Thanks @jenniferlee-dsva these are helpful. The latest files in Ryan's mural board don't reflect all of these decisions, so he or I can move forward with updating the screens for our developers.  

A couple of questions:

- For the 'view all articles in Xcategory' or 'tagged: Y' landing pages: the requirements say `truncate` the description after 200 characters but not how. Have you decided on how that will look?

- I think the results text should include how things are sorted, e.g. `Showing 1-10 of 89 results for "burials" sorted by relevance`.  What are your thoughts on that?

**Comment by jenniferlee-dsva** [comment]
I feel there are too many places where I have to reply to the same Qs for fear one of these venues get out of sync. 

Per slack: 
On dynamic search listings, the article blurb will be dynamic based on user query. We will not be adding any new functionality there.
On static 'view all articles in Xcategory/Y tag' pages, the article blurb will just pull from the intro text, truncating at 200 characters including spaces.

Beyond the 200 char limit, we don't have any other ux, so that would be good to have some design recommendations around:
Do we truncate and just add ... ? Do we truncate and add "Read more" ?

**Comment by andaleliz** [comment]
>I feel there are too many places where I have to reply to the same Qs for fear one of these venues get out of sync.

Yep, we are in a loop. I prefer to keep the conversation here and not buried in a Slack thread.

>Beyond the 200 char limit, we don't have any other ux, so that would be good to have some design recommendations around:
Do we truncate and just add ... ? Do we truncate and add "Read more" ?

Per my first comment today,  I recommend an ellipses. something like: <span aria-hidden="true">. . .</span> so screenreader users don't have to hear "dot dot dot" every time. Would need to validate that mark-up with a11y but that's a start.

Additionally, I think the results text should include how things are sorted, e.g. Showing 1-10 of 89 results for "burials" sorted by relevance. What are your thoughts on that?

**Comment by jenniferlee-dsva** [comment]
RE second bullet point - I am fine with what the designs show today and prefer not to complicate things with a change. Currently, search.gov results bold the relevant word/s in the listing, so I prefer to stay with that until that changes with the onsite search experience. 

LC search results screenshot - I'm okay with this design:
![image](https://user-images.githubusercontent.com/43381063/94484219-1139c980-01aa-11eb-9047-436cc23413d2.png)

**"All articles in: <category>" page** - I'm okay with this design (minus minor text updates -see requirements doc): 
![image](https://user-images.githubusercontent.com/43381063/94484456-742b6080-01aa-11eb-9bf6-85524277f1c6.png)


**"All articles tagged: <tag>"** page - I'm okay with this (minor minor text updates per requirements doc): 
![image](https://user-images.githubusercontent.com/43381063/94484557-a210a500-01aa-11eb-962d-5945b831f31a.png)

**Comment by MickinSahni** [comment]
@andaleliz to modify this ticket as needed for search results only. it'll include considerations for QA e.g. where components of the results come from

**Comment by andaleliz** [comment]
Alerted team via Slack, added screens to Mural board, complete.
